noir-projects/noir-contracts/contracts/nft_contract/src/test/transfer_to_private.nr

```diff
@@ -112,11 +74,9 @@ unconstrained fn transfer_to_private_failure_not_an_owner() {
     let (env, nft_contract_address, _, not_owner, token_id) = utils::setup_and_mint(/* with_account_contracts */ false);
 
     // We set random value for the commitment as the NFT owner check is before we use the value
-    let transfer_preparer_storage_slot_commitment = random();
+    let hiding_point_slot = random();
 
```

Comment:
This makes this test's correctness depend on the implementation of the contract (namely the order of the balance check and the point retrieval). I'd still set this up properly: that'll show that the test is doing everything correctly _except_ call from an account with balance, and hence the failure reason is clear.

Reply:
I did it like that to make the test faster. But I don't really have a strong opinion here so I did the change in https://github.com/AztecProtocol/aztec-packages/pull/9150/commits/0e9b164942cf76921a4c342c70a6676657e40ace